pkg/aggregator/aggregator.go

```diff
@@ -102,9 +102,9 @@ var (
 	aggregatorHostnameUpdate                   = expvar.Int{}
 
 	tlmFlush = telemetry.NewCounter("aggregator", "flush",
-		[]string{"type", "state"}, "Count of flush")
+		[]string{"metric_type", "state"}, "Count of flush")
```

Comment:
`metric_type:metrics` would be weird, how about:

```suggestion
		[]string{"data_type", "state"}, "Count of flush")
```

(open to other suggestions as well)

Reply:
Sounds good 👍 Thanks for the suggestion!